### PR TITLE
254-ULC: specify iterations

### PR DESCRIPTION
**Pull request by b00ris**

### CODE REVIEW

File: ideas/254-ultra-light-client/README.md [hunk 1]
```diff
@@ -63,14 +65,39 @@ Description:
...
 
-### Iteration 1
+### Iteration 2018-05-21 - 2018-06-04
 
-Goal Date: TBD
+Goal Date: 2018-06-04
+
+Description:
+
+- [ ] Fix [downloader N/M validation](https://github.com/status-im/go-ethereum/issues/51)
```
[the remark this answers is not in the document]
Comment (themue) [reply ideas/254-ultra-light-client/README.md]: Project is https://github.com/status-im/go-ethereum/projects/5

[the remark this answers is not in the document]
Comment (arnetheduck) [reply ideas/254-ultra-light-client/README.md]: so doesn't it make more sense to link to that here? what's the plan - track issues in both places?

[the remark this answers is not in the document]
Comment (b00ris) [reply ideas/254-ultra-light-client/README.md]: @arnetheduck, I'm going to use milestones for it. So, I was going to update it when I get a create milestones permission on GitHub for it.

[the remark this answers is not in the document]
Comment (arnetheduck) [reply ideas/254-ultra-light-client/README.md]: SGTM! just so we have a single source of information that's up to date!

**Pull request merged**